AppInsightsWithWebJob/Program.cs

```diff
@@ -37,14 +38,13 @@ static async Task Main(string[] args)
             var host = builder.Build();
             var cancellationToken = new WebJobsShutdownWatcher().Token;
```

Comment:
What is this?

Reply:
Left over from the original code. Will remove.